website/docs/r/databricks_virtual_network_peering.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+subcategory: "Databricks"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_databricks_virtual_network_peering"
+description: |-
+  Manages a Databricks Virtual Network Peering
+---
+
+# azurerm_databricks_virtual_network_peering
+
+Manages a Databricks Virtual Network Peering
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "databricks-vnet-peering"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "remote" {
+  name                = "remote-vnet"
+  resource_group_name = azurerm_resource_group.example.name
+  address_space       = ["10.0.1.0/24"]
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_databricks_workspace" "example" {
+  name                = "example-workspace"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku                 = "standard"
+}
+
+resource "azurerm_databricks_virtual_network_peering" "example" {
+  name                = "databricks-vnet-peer"
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_id        = azurerm_databricks_workspace.example.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+  allow_virtual_network_access  = true
+}
+
+resource "azurerm_virtual_network_peering" "remote" {
+  name                         = "peer-to-databricks"
+  resource_group_name          = azurerm_resource_group.example.name
+  virtual_network_name         = azurerm_virtual_network.remote.name
+  remote_virtual_network_id    = azurerm_databricks_virtual_network_peering.example.virtual_network_id
+  allow_virtual_network_access = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Databricks Virtual Network Peering resource. Possible valid values must begin with a letter or number, end with a letter, number or underscore, and may contain only letters, numbers, underscores, periods, or hyphens and must be between 1 and 80 characters in length. Changing this forces a new resource to be created.
```

Comment:
we don't tend to call out the validation requirements for string length etc here, the `validate` funcs handle this, so we can simplify this to:

```suggestion
* `name` - (Required) Specifies the name of the Databricks Virtual Network Peering resource. Changing this forces a new resource to be created.
```

Reply:
Fixed.

Reply:
Fixed.